libc/src/stdio/scanf_core/CMakeLists.txt

```
@@ -8,6 +8,22 @@ if(scanf_config_copts)
   list(PREPEND scanf_config_copts "COMPILE_OPTIONS")
 endif()
 
+
+list(APPEND file_deps
+  libc.hdr.types.FILE
+)
```

```suggestion
list(APPEND file_deps libc.hdr.types.FILE)
```
nit. could just be a single line.